### ISSUE TITLE
MIDI files with corrupt pitch values are not handled correctly

### ISSUE THREAD

**Issue by craffel** [craffel/pretty-midi#34]
Sometimes, `midi` reads in a MIDI file with has `NoteOnEvents` with `data[0] > 127`:

```Python
In [1]: import midi
In [2]: m = midi.read_midifile("data/clean_midi/mid/Celine Dion/That's The Way It Is.mid")
In [3]: for t in m:
   ...:     for e in t:
   ...:         if type(e) == midi.NoteOnEvent:
   ...:            if e.data[0] > 127:
   ...:                print e
   ...:
midi.NoteOnEvent(tick=4656, channel=7, data=[253, 75])
midi.NoteOnEvent(tick=0, channel=7, data=[254, 75])
midi.NoteOnEvent(tick=48, channel=7, data=[253, 0])
midi.NoteOnEvent(tick=0, channel=7, data=[254, 0])
```

This is because `midi` just loads in data via `ord` https://github.com/vishnubob/python-midi/blob/master/src/fileio.py#L94, so as long as the argument is a 8-bit char it will happily set a data value to a number more than 127.  So, this likely happens for other events too, not just `NoteOnEvents` `data[0]`.  This can create issues later on.  Either `midi` should raise an exception, we should raise an exception, or we should issue a warning and ignore those events with invalid data values.

**Comment by craffel** [comment]
This will be resolved by #99, because mido verifies that data bytes are less than 127: https://github.com/olemb/mido/blob/master/mido/midifiles.py#L282

**Comment by craffel** [comment]
Resolved in 8fbbdec5904e0766ed14433e39379704fa87454e.
